### PR TITLE
docs: added cw721 remarkables README.md

### PR DESCRIPTION
**Pull request by dadamu**
This PR added cw721 remarkables README.md, it only has difference on `Metadata` from `cw721-poap`.

### CODE REVIEW

File: contracts/cw721-remarkables/README.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+To easily interact with the contract you can use the `cw721-remarkables` script available [here](https://github.com/desmos-labs/contract-utils/tree/main/utils), 
+otherwise you can take a look at the supported messages in the following sections.
+
+## Instantiate Message
+Allows to initialize the contract. This message has the following parameters:
+* `name`: Name of the NFT contract;
+* `symbol`: Symbol of the NFT contract;
+* `minter`: Address who is the only one to be able to create new NFTs.
+
+An example of the instantiate message:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:13]:
```suggestion
An example of instantiate message:
```

File: contracts/cw721-remarkables/README.md [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+```
+
+## Execute Messages
+
+### TransferNft
+Allows to move a token to another account without triggering actions. This message has the following parameters:
+* `recipient`: Address where the token transfer to;
+* `token_id`: Id of the token which would be transferred.
+
+An example of the message to transfer nft:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:29]:
```suggestion
Here an example message to transfer nft:
```

File: contracts/cw721-remarkables/README.md [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+}
+```
+
+### SendNft
+Allows to move a token to another contract then trigger an action. This message has the following parameters:
+* `contract`: Contract address where the token transfer to;
+* `token_id`: Id of the token which would be transferred;
+* `msg`: Base64 encoded message to trigger on the receiver contract.
+
+An example of the message to send nft having a trigger message:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:45]:
```suggestion
Here an example message to send nft having a trigger message:
```

File: contracts/cw721-remarkables/README.md [hunk 4]
```diff
@@ -0,0 +1,535 @@
...
+        "contract": "desmos1......",
+        "token_id": "1-1",
+        "msg": "eyJleGVjdXRlX2V4YW1wbGUiOnt9fQ=="
+    }
+}
+```
+**Note** the msg is base64-encoded of `{"execute_example":{}}`, which is the sample execution message on the target contract. 
+
+### Approve
+Allows to give a user an access to transfer/send the token from the owner's account. This message has the following parameters:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:58]:
```suggestion
Allows a user to transfer/send the token from the owner's account. This message has the following parameters:
```

File: contracts/cw721-remarkables/README.md [hunk 5]
```diff
@@ -0,0 +1,535 @@
...
+{
+    "approve_all": {
+        "spender": "desmos1......",
+        "expires": null
+    }
+}
+```
+
+### RevokeAll
+Allows to remove a previously granted approval all permission. This message has the following parameters:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:155]:
```suggestion
Allows to remove a previously granted `ApproveAll` permission. This message has the following parameters:
```

File: contracts/cw721-remarkables/README.md [hunk 6]
```diff
@@ -0,0 +1,535 @@
...
+        "spender": "desmos1......",
+        "token_id": "1-1",
+        "expires": null
+    }
+}
+```
+
+### Revoke
+Allows to remove a previously granted approval. This message has the following parameters:
+* `spender`: Address who would be revoked the permission of the given token;
```
Comment (bragaz) [inline contracts/cw721-remarkables/README.md:102]:
```suggestion
* `spender`: Address at which the authorization of the given token is revoked;
```

File: contracts/cw721-remarkables/README.md [hunk 7]
```diff
@@ -0,0 +1,535 @@
...
+    "revoke": {
+        "spender": "desmos1......",
+        "token_id": "1-1"
+    }
+}
+```
+
+### ApproveAll
+Allows to give all the tokens transferring/sendind tokens approval to an operator from the owner's account. This message has the following parameters:
+* `operator`: Address who is assigned to have all the tokens approvals in the owner's account;
```
Comment (bragaz) [inline contracts/cw721-remarkables/README.md:117]:
```suggestion
* `operator`: Address to which all token approvals are assigned in the owner account;
```

File: contracts/cw721-remarkables/README.md [hunk 8]
```diff
@@ -0,0 +1,535 @@
...
+    "burn": {
+        "token_id": "1-1"
+    }
+}
+```
+
+## Query Messages
+
+### OwnerOf
+Returns the owner of the given token, error if token does not exist. This message has the following parameters:
```
Comment (bragaz) [inline contracts/cw721-remarkables/README.md:206]:
```suggestion
Returns the owner of the given token, or an error if the token does not exist. This message has the following parameters:
```

File: contracts/cw721-remarkables/README.md [hunk 9]
```diff
@@ -0,0 +1,535 @@
...
+            "expiration": {
+                "never": {}
+            }
+        },
+    ]
+}
+```
+
+### Approval
+Returns an access owned by the given spender to the given token. This message has the following parameters:
```
Comment (bragaz) [inline contracts/cw721-remarkables/README.md:248]:
```suggestion
Returns an approval owned by the given spender to the given token. This message has the following parameters:
```

File: contracts/cw721-remarkables/README.md [hunk 10]
```diff
@@ -0,0 +1,535 @@
...
+        "spender": "desmos1......",
+        "expiration": {
+            "at_height": 1000
+        }
+    } 
+}
+```
+
+### Approvals
+Returns approvals that a token has. This message has the following parameters:
```
Comment (bragaz) [inline contracts/cw721-remarkables/README.md:277]:
```suggestion
Returns all the approvals that a token has. This message has the following parameters:
```

**Review by manu0466 — APPROVED** [review]

File: contracts/cw721-remarkables/README.md [hunk 11]
```diff
@@ -0,0 +1,535 @@
...
+```
+
+## Query Messages
+
+### OwnerOf
+Returns the owner of the given token, or an error if the token does not exist. This message has the following parameters:
+* `token_id`: Id of the target token;
+* `include_expired`: Trigger to filter out expired approvals, unset or false will exclude expired approvals.
+
+Here Here an example message to query the owner of the given token:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:210]:
```suggestion
Here an example message to query the owner of the given token:
```

File: contracts/cw721-remarkables/README.md [hunk 12]
```diff
@@ -0,0 +1,535 @@
...
+}
+```
+
+### Approval
+Returns an approval owned by the given spender to the given token. This message has the following parameters:
+* `token_id`: Id of the target token;
+* `spender`: Address who has the sending/transferring access to the given token;
+* `include_expired`: Trigger to filter out expired approvals, unset or false will exclude expired approvals.
+
+Here Here an example message to query the approval of the given token by a spender:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:253]:
```suggestion
Here an example message to query the approval of the given token by a spender:
```

File: contracts/cw721-remarkables/README.md [hunk 13]
```diff
@@ -0,0 +1,535 @@
...
+    } 
+}
+```
+
+### Approvals
+Returns all the approvals that a token has. This message has the following parameters:
+* `token_id`: Id of the target token;
+* `include_expired`: Trigger to filter out expired approvals, unset or false will exclude expired approvals.
+
+Here Here an example message to query the approvals of the given token:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:281]:
```suggestion
Here an example message to query the approvals of the given token:
```

File: contracts/cw721-remarkables/README.md [hunk 14]
```diff
@@ -0,0 +1,535 @@
...
+```
+
+### AllOperators
+Lists all operators that can access all of the owner's tokens. This message has the following parameters:
+* `owner`: Address of the owner to be queried.
+* `include_expired`: Trigger to filter out expired approvals, unset or false will exclude expired approvals;
+* `start_after`: Position in address where tokens start after;
+* `limit`: Limitation to list the number of operators, if unset would be 10 and the maximum is 100.
+
+Here Here an example message to query the operators of the given owner:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:324]:
```suggestion
Here an example message to query the operators of the given owner:
```

File: contracts/cw721-remarkables/README.md [hunk 15]
```diff
@@ -0,0 +1,535 @@
...
+```json
+{
+    "count": 1000
+}
+```
+
+### ContractInfo
+Returns top-level metadata about the contract.
+
+Here Here an example message to query the contract info of the contract:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:382]:
```suggestion
Here an example message to query the contract info of the contract:
```

File: contracts/cw721-remarkables/README.md [hunk 16]
```diff
@@ -0,0 +1,535 @@
...
+    "name": "test_name",
+    "symbol": "test_symbol"
+}
+```
+
+### NftInfo
+Returns metadata about one particular token. This message has the following parameters:
+* `token_id`: Id of the target token.
+
+Here Here an example message to query the info of the given token:
```
Comment (manu0466) [inline contracts/cw721-remarkables/README.md:401]:
```suggestion
Here an example message to query the info of the given token:
```

**Comment by manu0466** [comment]
Just some errors to fix and it's good to me

**Review by manu0466 — APPROVED** [review]

**Review by bragaz — APPROVED** [review]

**Pull request merged**